client/src/pages/SubmissionExport.vue

```diff
@@ -0,0 +1,55 @@
+<template>
```

Comment:
Is there any guard preventing me from going to this page directly and accessing the export ?  I looks like all the permissions checking is done when the buttons are made, but not checked here.  (If its there and I just missed it, feel free to ignore it)

Reply:
Good spot. It currently checks to see if the user has a role on the submission or has editor-level permissions, but it doesn't prevent a reviewer from accessing this page after the review has been closed. I'll see what I can do to make this consider the submission's status as well.

Reply:
The rules for authorizing access to the Export page have been implemented.